### PR TITLE
add cloudinit doc link

### PR DESCRIPTION
**Pull request by glekner**
Screen Shot 2020-10-28 at 15 52 44

### CODE REVIEW

**Comment by glekner** [comment]
@matthewcarleton is the placement and text of the doc btn ok?

**Comment by matthewcarleton** [comment]
I'd like to pull in @yfrimanm here. IMO we should provide some inline help and then send them off to the docs if they want to learn more.

**Comment by yfrimanm** [comment]
@glekner WDYT?
image


Cloud-init
image

**Comment by glekner** [comment]
Updated to a helper with a new text with the help of Tomas

image

@yfrimanm @matthewcarleton

**Comment by yfrimanm** [comment]
Thank you, LGTM!

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/cloud-init-tab/cloud-init-tab.tsx [hunk 1]
```diff
@@ -302,7 +307,39 @@ const CloudInitTabComponent: React.FC<ResultTabComponentProps> = ({
       )}
       <Form>
         <Title headingLevel="h5" size="lg">
-          Cloud-init
+          Cloud-init{' '}
+          <Popover
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/cloud-init-tab/cloud-init-tab.tsx:311]: can you please move this to a standalone component as this is component is quite already big?

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/cloud-init-tab/cloud-init-tab.tsx [hunk 2]
```diff
@@ -302,7 +307,39 @@ const CloudInitTabComponent: React.FC<ResultTabComponentProps> = ({
...
+                </StackItem>
+                <StackItem>
+                  <div className="text-muted">
+                    cloud-init is already configured in cloud images of Fedora and RHEL
+                  </div>
+                </StackItem>
+                <StackItem>
+                  <ExternalLink
+                    text="Learn more"
+                    href="https://cloudinit.readthedocs.io/en/latest/index.html"
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/cloud-init-tab/cloud-init-tab.tsx:328]: extracting it to constant would be probably better

**Review by suomiy — COMMENTED** [review]
looks good

**Comment by glekner** [comment]
created a new component @suomiy

**Pull request merged**